build/webapp-shared.js

```diff
@@ -396,7 +396,8 @@ WebappShared.prototype.execute = function(options) {
 
 function execute(config) {
   var gaia = utils.gaia.getInstance(config);
-  gaia.rebuildWebapps.forEach(function(webapp) {
+  config.GAIA_APPDIRS.split(' ').forEach(function(appDir) {
```

Comment:
Because we don't have rebuildWebapps in getInstance anymore.

You might also ask why we don't attach rebuildAppDirs to config through rebuild.js anymore?
The answer is, we want to use makefile's logic to decide what app's content has changed and rebuild them instead of using rebuild.js. We should detect whether to regenerate makefile in bug 1154583.